riscv/insns/sspopchk_x1.h

```diff
@@ -0,0 +1,12 @@
+#include "zicfiss.h"
+require_extension(EXT_ZICFISS);
+
+bool xSSE = false;
+SET_XSSE(xSSE);
+
+if (xSSE) {
+  reg_t ssp = STATE.ssp->read();
+  const auto new_ssp = ssp + xlen / 8;
+  check_ssp(RA, ssp);
+  STATE.ssp->write(new_ssp);
+}
```

Comment:
If we used `RS1` instead of `RA` then we can just include "sspopchk_x1.h" in "sspopchk_x5.h".